courses/machine_learning/deepdive2/recommendation_systems/labs/als_bqml_hybrid.ipynb

```diff
@@ -0,0 +1,956 @@
+{
```

Comment:
<p>We only need PROJECT here since the others aren't used in this lab.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/753/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done</p>